src/components/templates/ScreeningRoom/useScreeningRoom.ts

```diff
@@ -0,0 +1,148 @@
+import { useState, useMemo, useCallback } from "react";
+import Fuse from "fuse.js";
+
+import { DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT } from "settings";
+
+import { isTruthy } from "utils/types";
+import { screeningRoomVideosSelector } from "utils/selectors";
+
+import { isLoaded, useFirestoreConnect } from "hooks/useFirestoreConnect";
+import { useDebounceSearch } from "hooks/useDebounceSearch";
+import { useSelector } from "hooks/useSelector";
+
+const emptyScreeningRoomVideosArray: never[] = [];
+
+export const useConnectScreeningRoomVideos = (screeningRoomVenueId: string) => {
+  useFirestoreConnect([
+    {
+      collection: "venues",
+      doc: screeningRoomVenueId,
+      subcollections: [{ collection: "screeningRoomVideos" }],
+      storeAs: "screeningRoomVideos",
+    },
+  ]);
+};
+
+export const useScreeningRoomVideos = (screeningRoomVenueId: string) => {
+  useConnectScreeningRoomVideos(screeningRoomVenueId);
+
+  const screeningRoomVideos = useSelector(screeningRoomVideosSelector);
+
+  console.log({ screeningRoomVenueId, screeningRoomVideos });
```

Comment:
log

Reply:
Sorry, it was an outdated code. It's gone now